src/Modal/Modal.tsx

```diff
@@ -19,6 +19,7 @@ export type ModalProps = React.HTMLAttributes<HTMLDivElement> &
     footer?: boolean
     acceptText?: string
     cancelText?: string
+    backdropClickCancels?: boolean
```

Comment:
Could we rename this to something like "closeOnBlur"?

Reply:
Couldn't decide what to name it, that's why I left this PR open. Change is committed now!